syntax.Rmd

```diff
@@ -465,26 +465,49 @@ do_something_very_complicated("that", requires, many, arguments,
 
 As described under [Named arguments](#argument-names), you can omit the argument names
 for very common arguments (i.e. for arguments that are used in almost every
-invocation of the function). Short unnamed arguments can also go on the same
-line as the function name, even if the whole function call spans multiple lines.
+invocation of the function). If this introduces a large disparity between the line lengths, you may want to supply names anyway:
 
 ```{r, eval = FALSE}
-map(x, f,
+# Good
+map(
+  x,
+  f,
+  extra_argument_a = 10,
+  extra_argument_b = c(1, 43, 390, 210209)
+)
+
+# Also good
+map(
+  x = x,
+  f = f,
   extra_argument_a = 10,
   extra_argument_b = c(1, 43, 390, 210209)
 )
 ```
 
-You may also place several arguments on the same line if they are closely
-related to each other, e.g., strings in calls to `paste()` or `stop()`. When
-building strings, where possible match one line of code to one line of output.
+You may also place multiple unnamed arguments on the same line if they are closely
+related to each other, e.g., strings in calls to `paste()` or arguments to `sprintf()`. When
+building strings, match one line of code to one line of output where possible.
 
 ```{r, eval = FALSE}
 # Good
 paste0(
   "Requirement: ", requires, "\n",
   "Result: ", result, "\n"
 )
+sprintf(
+  "formatting string %s%s%s%s",
+  a, b, c, d
+)
+
+# Ok
+sprintf(
+  "formatting string %s%s%s%s",
+  a,
+  b,
+  c,
+  d
+)
 
 # Bad
 paste0(
```

Comment:
Maybe keep the `# Good` and `# Bad` `paste0()` examples together? Helps with skimming and comparison i think